app/modules.go

```diff
@@ -96,95 +97,95 @@ func (app *App) setupModuleManager(skipGenesisInvariants bool) error {
 	app.manager, err = module.NewManager([]module.VersionedModule{
 		{
 			Module:      genutil.NewAppModule(app.AccountKeeper, app.StakingKeeper, app.BaseApp.DeliverTx, app.txConfig),
-			FromVersion: v1, ToVersion: v2,
+			FromVersion: v1, ToVersion: appconsts.LatestVersion,
```

Comment:
for things that are not being removed, I think we should just use the latest version. when they get removed or replaced, then we add a ToVersion.

open to not donig this